client/src/components/main/Favorite/FavoriteShops.scss

```diff
@@ -0,0 +1,86 @@
+.saved-shops {
+  width: 85%;
+  height: 850px;
+  background: #efefef;
```

Comment:
Just as a check, if these colors are intended to align with our theme, I would suggest utilizing the variables from variables.scss to stay consistent sitewide

Reply:
so the container color is slightly different from the background color, so the favorite shops are all listed in pannel (just try to follow the design of trulia) if you dont like it, we can always change it

Reply:
that's cool, just checking. I vary some things based on design judgments as well

Reply:
For minor color edits I'd recommend using [the build in sass color modifiers](https://devhints.io/sass) on the variable you want to tweak

Reply:
Also the color you picked here is nearly identical to the `$surface` color variable